Docs/source/running_cpp/parameters.rst

```diff
@@ -1433,7 +1433,7 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
 
 * ``<diag_name>.fields_to_plot`` (list of `strings`, optional)
     Fields written to output.
-    Possible values: ``Ex`` ``Ey`` ``Ez`` ``Bx`` ``By`` ``Bz`` ``jx`` ``jy`` ``jz`` ``part_per_cell`` ``rho`` ``F`` ``part_per_grid`` ``part_per_proc`` ``divE`` ``divB`` and ``rho_<species_name>``, where ``<species_name>`` must match the name of one of the available particle species.
+    Possible values: ``Ex`` ``Ey`` ``Ez`` ``Bx`` ``By`` ``Bz`` ``jx`` ``jy`` ``jz`` ``part_per_cell`` ``rho`` ``F`` ``part_per_grid`` ``divE`` ``divB`` and ``rho_<species_name>``, where ``<species_name>`` must match the name of one of the available particle species.
```

Comment:
I don't think `part_per_proc` is a possible field value so I've removed it from the list.

Reply:
There is another mention of this in `Examples/Tests/Langmuir/README.md` it seems. Can you remove this as well? :)

Reply:
@EZoni @NeilZaim does `rho_<species_name>` need a no-particle branch as well?

Reply:
@ax3l If I understand correctly, your question is what happens if we try to output `rho_<species_name>` when there is 0 particle species? In this case I think we will simply get an abort with an error message stating that `<species_name>` must match one of the particle species.

Reply:
Thanks for clarifying my confusion. Makes total sense.

Reply:
Looks like `part_per_grid` is removed as well in AMReX & WarpX

Reply:
Hm, now I wonder if `part_per_proc` is just missing in this list: https://github.com/ECP-WarpX/WarpX/blob/dde26c13e9a54dbae396b02a3daad7f09652747d/Source/Diagnostics/FullDiagnostics.cpp#L401-L404

Reply:
I'm actually confused what the `PartPerProc` functor would actually entail.  Say you want to plot that field, what does it look like?  I think it could be more appropriate as a reduced diagnostic, where you have a little more flexibility about the output format.  I just realized that in principle, `PartPerProc` exists already in the costs reduced diagnostics, if you used heuristic with particle weight of one and field weight of zero (and you could set the load balance interval to by large).  This would contain all the information needed to compute particles per proc.